package com.example;

import com.example.infraestructure.database.person.PersonRepository;
import org.junit.Test;
import org.junit.runner.RunWith;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.junit4.SpringRunner;

import javax.sql.DataSource;

import java.sql.SQLException;

import static org.junit.Assert.assertThat;
import static org.junit.Assert.assertTrue;

@RunWith(SpringRunner.class)
@SpringBootTest
public class DemoApplicationTests {

    @Autowired
    private DataSource dataSource;

    @Autowired
    private PersonRepository repository;

    @Test
    public void test() {

        try {
            System.out.println(dataSource.getConnection());
            System.out.println(repository);

        } catch (SQLException e) {
            e.printStackTrace();
        }
    }

}
